src/backend/services/carValueService.ts

```diff
@@ -0,0 +1,43 @@
+interface CarValue {
+  id: number;
+  model: string;
+  year: number;
+}
```

Comment:
Could add a value: to this
```suggestion
interface CarValue {
  id: number;
  model: string;
  year: number;
  value: number
}
```

then when creating your array you could use the calculate function to create the car's value globally and push the car's calculated value along with it's details to your array using your createCarValue function